README.md

```diff
@@ -163,29 +167,24 @@ yarn nx lint core
 
 #### Manage task cache
 
-When running the Build, Lint or Test commands, **Nx** can use its [cache mechanism](https://nx.dev/concepts/how-caching-works).
-For some reason, it can be useful to clean the cache to investigate issue, this can be done via the following command:
+When running the Build, Lint, or Test commands, **Nx** can use its [cache mechanism](https://nx.dev/concepts/how-caching-works).
+If for some reason it is useful to clean the cache to investigate an issue, this can be done via the following command:
```

Comment:
Not sure about the "if"

Reply:
is it really always useful ? The if is not shocking here

Reply:
I modified the sentence to: "In some cases, it may be useful to clear the cache to investigate an issue."